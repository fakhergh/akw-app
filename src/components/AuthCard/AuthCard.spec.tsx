import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import { AuthCard } from '@/components/AuthCard/AuthCard';

describe('<AuthCard/>', () => {
    it('Should matches snapshot', () => {
        const { container } = render(<AuthCard />);
        expect(container).toMatchSnapshot();
    });
});
