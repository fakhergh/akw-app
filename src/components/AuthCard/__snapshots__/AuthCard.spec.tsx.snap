// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<AuthCard/> > Should matches snapshot 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-15d2lr-MuiPaper-root-MuiCard-root"
    style="--Paper-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);"
  />
</div>
`;
